js/reload_controller.js

```diff
@@ -22,11 +22,15 @@ ReloadController = function()
     reloadUnpinnedOnly: false,
     reloadAllRight: false,
     reloadAllLeft: false,
+    closeAllRight: false,
+    closeAllLeft: false,
     reloadStartup: 'none',
-    bypassCache: false
+    bypassCache: false,
+	buttonDefaultAction: "window"
```

Comment:
```suggestion
  buttonDefaultAction: 'window'
```
nit: space instead of tab, and single quotes instead of double

Reply:
updated in my fork